openmessaging-api/src/main/java/io/openmessaging/consumer/Consumer.java

```diff
@@ -176,6 +228,24 @@
      */
     Message receive(long timeout);
 
+    /**
+     * Receives the next message from the which bind queue,partition and receiptId of this consumer in pull model.
+     * <p>
+     * This call blocks indefinitely until a message is arrives, the timeout expires, or until this {@code PullConsumer}
+     * is shut down.
+     *
+     * @param queueName receive message from which queueName in Message Queue.
+     * @param partitionId receive message from which partition in Message Queue.
+     * @param receiptId  receive message from which receipt position in Message Queue.
+     * @param timeout receive message will blocked at most <code>timeout</code> milliseconds.
+     * @return the next message received from the bind queues, or null if the consumer is concurrently shut down.
+     * @throws OMSSecurityException when have no authority to receive messages from this queue.
+     * @throws OMSTimeOutException when the given timeout elapses before the send operation completes.
+     * @throws OMSRuntimeException when the {@code Producer} fails to send the message due to some internal error.
+     */
+    Message receive(String queueName,@Optional int partitionId, long receiptId, long timeout);
```

Comment:
it would be nice if you can put this anotation above this method.

Reply:
Okay,I have already finished it.